langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -111,8 +111,8 @@ isolated function findAllGroupsImpl(RegExp reExp, string str, int startIndex = 0
     name: "findAllGroups"
 } external;
 
-public isolated function matchAt(RegExp 're, string str, int startIndex = 0) returns Span? {
-    SpanAsTupleType? resultArr = matchAtImpl('re, str, startIndex);
+public isolated function matchAt(RegExp re, string str, int startIndex = 0) returns Span? {
```

Comment:
Shall we a few test cases for named argument expressions?

Reply:
Sure

Reply:
Added 3e5e26ad83b7c016fffa0258ba7092b88024d2fd